docs/en/Storage.md

```diff
@@ -1,132 +1,261 @@
-# Storage
+# Storage Types
 
-The platform has a range of different types of storage meant for a variety of
-use-cases, so this storage section applies whether you're experimenting,
-creating pipelines, or publishing.
+The platform provides two main types of storage:
 
-At the surface, there are two kinds of storage
+- Disk (also called Volumes on the Notebook Server creation screen)
+- Bucket ("Blob" or S3 storage, provided through MinIO)
 
-- Disks (also called Volumes)
-- Buckets (S3 or "Blob" storage)
+Depending on your use case, either may be most suitable:
+
+|   Type |                                                                                                         Simultaneous Users |                                                   Speed | Total size               |
+| -----: | -------------------------------------------------------------------------------------------------------------------------: | ------------------------------------------------------: | ------------------------ |
+|   Disk |                                       One compute instance at a time (one notebook server, step in kubeflow pipeline, ...) |                        Fastest (throughput and latency) | <=512GB total per drive  |
```

Comment:
I think "One compute instance at a time (one notebook server, step in kubeflow pipeline, ...)" is too jargon-y. Maybe replace "Compute Instance" with Notebook server.

If someone is *capable* of figuring out how to tell their kubeflow pipeline the name of a PVC, they will probably figure out on their own that that doesn't work!

So maybe just

"Only one machine/notebook server at a time."